### DISCUSSION TITLE
Oyun oynama vidyo izleme gibi özelliklerin gelmesi ve kullanımı kolay cüzdan

### DISCUSSION THREAD

**Discussion by Bayramp** [IBAX-io/go-ibax#469] [Ideas]
### Clear and concise description of the problem

Oyun oynama puan arttırma vidyo izleme puan arttırma gibi özelliklerin gelmesi ve kullanımı kolay cüzdan oluşumu yapılması daha ilgi çekici olur. Ayrıca nezaman hangi borsalarda çıkacacağı veya tüm borsalarda bi andamı gireceği gibi konuların bizlere daha açık ve net anlatılması 

### Suggested solution

Oyun oynama puan arttırma vidyo izleme puan arttırma gibi özelliklerin gelmesi ve kullanımı kolay cüzdan oluşumu yapılması daha ilgi çekici olur. Ayrıca nezaman hangi borsalarda çıkacacağı veya tüm borsalarda bi andamı gireceği gibi konuların bizlere daha açık ve net anlatılması 

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by Emircangzc** [comment]
This is wrong
